infrastructure/terragrunt/aws/load-balancer/waf.tf

```diff
@@ -22,7 +22,17 @@ resource "aws_wafv2_web_acl" "wordpress_waf" {
     priority = 1
 
     override_action {
-      none {}
+      dynamic "none" {
+        for_each = var.enable_waf ? [""] : []
+        content {
+        }
+      }
+
+      dynamic "count" {
+        for_each = var.enable_waf == false ? [""] : []
+        content {
+        }
+      }
```

Comment:
I really like this pattern, but a minor comment - could we switch to be consistent in how we test for `true/false`?   I'm not fussed either way:

```hcl
      dynamic "none" {
        for_each = var.enable_waf == true ? [""] : []
        content {
        }
      }

      dynamic "count" {
        for_each = var.enable_waf == false ? [""] : []
        content {
        }
      }
``` 

or

```hcl
      dynamic "none" {
        for_each = var.enable_waf ? [""] : []
        content {
        }
      }

      dynamic "count" {
        for_each = !var.enable_waf ? [""] : []
        content {
        }
      }
```